src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec

```diff
@@ -13,5 +13,6 @@
   </metadata>
   <files>
       <file src="NuGet.exe" target="tools" />
+      <file src="NuGet.pdb" target="tools" />
```

Comment:
It didn't before and won't after the change. nuget.exe is smart enough not to include symbols into regular package :smiley_cat: